landoapi/transplants.py

```diff
@@ -962,21 +962,16 @@ def get_parsed_diffs(
     }
 
 
-def assess_stack_state(
+def build_stack_assessment_state(
```

Comment:
I realize this could probably be a little cleaner (maybe even a classmethod/constructor on `StackAssessmentState` directly?) but I'd like to defer making it optimal for now, since the actual goal is to make `create_state` an integrated test fixture, and we have already discussed doing some more cleanups after more of the checks/hooks migration work is complete.

Reply:
I had a thought that the `StackAssessmentState` could become a `StackAssessor`, and we could turn `run_landing_checks` into a `.run_assessment() -> StackAssessment` method. Just a thought for future improvements, it would make it more obvious that the `run_landing_checks` function does perform mutations on the `StackAssessmentState` object.